package.json

```diff
@@ -58,6 +58,7 @@
     "body-parser": "^1.14.1",
     "bucket-assets": "^1.0.2",
     "cheerio": "^0.19.0",
+    "coffee-loader": "^0.9.0",
     "coffeescript": "1.11.1",
```

Comment:
The issue is that `coffee-script` isn't in the dependency list -- since they renamed the package there might be some mismatch. `yarn add coffee-script@1.11.1` should fix things.

Reply:
Ohh I see. In that case I think updating the task to 

`"task": "node -r coffeescript/register -r @babel/register -r dotenv/config"`

should do it?

Reply:
Yup! i think that should work

Reply:
Cool, thanks!